# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dietlibc/sys_nmln.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./include/sys/utsname.h.vanilla	2006-07-19 13:08:26.000000000 +0200
+++ ./include/sys/utsname.h	2006-07-19 13:08:54.000000000 +0200
@@ -14,6 +14,8 @@
    to a value != 0 to activate this entry.  */
 #define _UTSNAME_DOMAIN_LENGTH _UTSNAME_LENGTH
 
+#define SYS_NMLN _UTSNAME_LENGTH
+
 struct utsname {
   /* Name of the implementation of the operating system.  */
   char sysname[_UTSNAME_LENGTH];
